roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install -r {{ taca_ngi_dest }}/requirements.txt"
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install ."
+  args: 
+    chdir: {{ taca_ngi_dest }}
+  tags: taca
+
+-name: Create TACA delivery configs directory
+ file: path="{{ ngi_pipeline_conf }}/TACA" state=directory mode=g+s
+ tags: taca
+
+- set_fact: 
+    site: "upps"
+    site_full: "uppsala"
+    ngi_pipeline_site_path: {{ ngi_pipeline_upps_path }}
+    ngi_pipeline_site_delivery: {{ ngi_pipeline_upps_path }}
+  tags: taca
+
+- name: Deploy site_taca_delivery.yml config
+  template: src="{{ site }}_taca_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/{{ site }}_taca_delivery.yml"
```

Comment:
`src` should be `site_taca_delivery.yml.j2`, because we do not want to expand `site` in this case.